src/routes/EditCustomFields.js

```diff
@@ -0,0 +1,29 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import { EditCustomFieldsSettings } from '@folio/stripes/smart-components';
+
+class EditCustomFields extends Component {
+  static propTypes = {
+    history: PropTypes.shape({
+      push: PropTypes.func.isRequired,
+    }).isRequired,
+  };
+
+  redirectToView = () => {
+    this.props.history.push('/settings/users/custom-fields');
+  }
+
+  render() {
+    return (
+      <EditCustomFieldsSettings
+        backendModuleName="users"
+        entityType="user"
+        entityTypeTranslationId="ui-users.meta.title"
```

Comment:
`entityTypeTranslationId` property is not needed. I forgot to remove it from here after I removed it from `EditCustomFieldsSettings`  and `ViewCustomFieldsSettings`